etc/atom.profile

```diff
@@ -23,6 +25,7 @@ notv
 novideo
 protocol unix,inet,inet6,netlink
 seccomp
+net none
```

Comment:
does allowing inet(6) protocols still make sense with "net none"?

Reply:
No, it would also break the extension/themes store

Reply:
we could put it optional :)

Reply:
optional net none (commented out)